### DISCUSSION TITLE
v-table filter system

### DISCUSSION THREAD

**Discussion by hypericon** [vuetifyjs/vuetify#12519] [Ideas]
hello, 

I don't seem to find something alike: the filter input above the table component is cool, but what if I would like to filter, let's say, with checkboxes for specific table values?

ty!

**Comment by sugoidesune** [comment]
It is the very first example in  the docs:
https://vuetifyjs.com/en/components/data-tables/#custom-filter
Inside the header add a filter property with a function:
![image](https://user-images.githubusercontent.com/20317691/97770560-aa0a8e80-1b34-11eb-925f-af4f4a4423a9.png)

**Reply by hypericon** [reply]
Yes i already did it.
This wasn’t exactly what I meant indeed: but I realized it this is a matter of filtering the json or an api call: I don’t want to use an input text but the checkbox or an Amazon system with chips
